src/ray/core_worker/store_provider/memory_store/memory_store.h

```diff
@@ -217,6 +220,10 @@ class CoreWorkerMemoryStore {
   /// Number of object store memory used by this memory store. (It doesn't include plasma
   /// store memory usage).
   int64_t used_object_store_memory_ GUARDED_BY(mu_) = 0;
+
+  std::function<std::shared_ptr<RayObject>(const RayObject &object,
+                                           const ObjectID &object_id)>
+      object_allocator_;
```

Comment:
Document that this "object_allocator_" is used to allow language frontend to allocate the objects in the memory store.